.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "Mobile-Expensify"]
```

Comment:
I wonder if we can come up with a less confusing name for the submodule.. Maybe `OldApp` 🤔 

cc @Julesssss

Reply:
I wanted to do that, but unfortunately the OldDot repo has some paths that use `Mobile-Expensify` in order to eg. attach `app` folder, so I decided to leave it as it is 😢

Reply:
Fair enough.